edx_sga/sga.py

```diff
@@ -55,6 +56,7 @@ class StaffGradedAssignmentXBlock(XBlock):
     """
     has_score = True
     icon_class = 'problem'
+    STUDENT_FILEUPLOAD_MAX_SIZE = 4 * 1000 * 1000  # 4 MB
```

Comment:
@amir-qayyum-arbisoft I am not sure about adding this here. If the `env.py` has a limit then it should be enforced, otherwise, we can just not check the file size on the client. Or is having a default in the module a requirement?